examples/lines_bars_and_markers/bar_label_demo.py

```diff
@@ -21,25 +21,25 @@
 # Define the data
 
 N = 5
-menMeans = (20, 35, 30, 35, -27)
-womenMeans = (25, 32, 34, 20, -25)
-menStd = (2, 3, 4, 1, 2)
-womenStd = (3, 5, 2, 3, 3)
+coffee_means = (20, 25, -10, 32, 10)
+tea_means = (30, 13, -14, 21, 17)
+coffee_std = (3, 2, 4, 1, 2)
+tea_std = (4, 3, 2, 3, 5)
 ind = np.arange(N)    # the x locations for the groups
-width = 0.35       # the width of the bars: can also be len(x) sequence
+width = 0.25       # the width of the bars: can also be len(x) sequence
 
 ###############################################################################
 # Stacked bar plot with error bars
 
 fig, ax = plt.subplots()
 
-p1 = ax.bar(ind, menMeans, width, yerr=menStd, label='Men')
-p2 = ax.bar(ind, womenMeans, width,
-            bottom=menMeans, yerr=womenStd, label='Women')
+p1 = ax.bar(ind, coffee_means, width, yerr=coffee_std, label='Coffee')
+p2 = ax.bar(ind, tea_means, width,
+            bottom=coffee_means, yerr=tea_std, label='Tea')
 
 ax.axhline(0, color='grey', linewidth=0.8)
 ax.set_ylabel('Scores')
-ax.set_title('Scores by group and gender')
+ax.set_title('Scores by group and their beverage choices')
```

Comment:
I just realized that the example has been semantically weird already before. The stacked bar suggests that the quantity is summed up from both subgroups, and the errorbar indicates that within each subgroup the quantity is averaged. I cannot come up with a reasonable quantity for "Scores" that would actually work this way.

Second, before there were groups and inside these groups we distinguished by gender. This is something everybody is familiar with. Splitting a group up by beverage choice and scoring these subgroups is possible but an even more weird experiment.

I think we have two choices:
a) Come up with something completely new
b) Or accept that the plot is not semantically meaningful, merge and do something a) later.

@story645 what do you think?

Reply:
Ugh, I want to be okay with merging something that's not semantically meaningful 'cause @SourajitaDewasi worked so much on this, but I hear you that the semantics being off might be a distraction that will make the example harder to follow. 
Keeping it simple, maybe change the underlying pivot table to

(caffeinated, tea), (decaffeinated, tea), (caffeinated, coffee), (decaffeinated, tea) and the y axis is like #people who prefer this drink so the example is basically a made up count of what type of drink folks prefer. 

Alternatively, palmer penguins  is being used a lot to illustrate aggregated visualizations, and we can probably keep this example pretty small if what we store in the example is the pivot table made from it https://allisonhorst.github.io/palmerpenguins/

Reply:
I would be fine with merging as-is as an improvement, with the suggestion below, and have a follow up either a) removing the example, or b) reworking it to include penguins or other good ideas.

Reply:
Ok, let's take this as is (suggestion below committed) so that we can git this in 3.6. And let's think about how we can further improve later.